### PR TITLE
verbose mode (light for now)

### PR DESCRIPTION
**Pull request by volodia99**
may be useful for future works

### CODE REVIEW

File: nonos/logging.py [hunk 1]
```diff
@@ -17,3 +19,18 @@ def print_err(message):
     https://github.com/neutrinoceros/idefix_cli
     """
     rprint(f":bone: [bold white on red]Error[/] {message}", file=sys.stderr)
+
+
+def logging_verbose(verbose):
```
Comment (neutrinoceros) [inline nonos/logging.py:24]: This took me much too long to decipher
```suggestion
def logging_verbose(verbose:int):
```

File: tests/test_plotting.py [hunk 2]
```diff
@@ -35,7 +35,8 @@ def test_plot_simple(argv, simulation_dir, capsys, tmp_path):
 
     out, err = capsys.readouterr()
     assert err == ""
-    assert re.match(r"Operation took \d+.\d\ds\n", out)
+    # assert re.match(r"Operation took \d+.\d\ds\n", out)
```
Comment (neutrinoceros) [inline tests/test_plotting.py:38]:
```suggestion
```

File: tests/test_plotting.py [hunk 3]
```diff
@@ -57,7 +59,8 @@ def test_plot_simple_corotation(simulation_dir, capsys):
     ret = main(["-cor", "-dir", str(simulation_dir)])
 
     out, err = capsys.readouterr()
-    assert re.match(r"Operation took \d+.\d\ds\n", out)
+    assert out == ""
+    # assert re.match(r"Operation took \d+.\d\ds\n", out)
```
Comment (neutrinoceros) [inline tests/test_plotting.py:63]:
```suggestion
```

File: nonos/main.py [hunk 4]
```diff
@@ -863,6 +863,8 @@ def plot(
                 cbar = plt.colorbar(im, orientation="vertical")
                 cbar.set_label(self.title)
 
+        logging.debug("---> done")
```
[the remark this answers is not in the document]
Comment (volodia99) [reply nonos/main.py:866]: What do you mean by 'actual name of the task' ? Should we print only one cpu ?

[the remark this answers is not in the document]
Comment (volodia99) [reply nonos/main.py]: okok

File: nonos/logging.py [hunk 5]
```diff
@@ -17,3 +19,17 @@ def print_err(message):
     https://github.com/neutrinoceros/idefix_cli
     """
     rprint(f":bone: [bold white on red]Error[/] {message}", file=sys.stderr)
+
+
+def setup_logging(verbose: int):
+    levels = ["WARNING", "INFO", "DEBUG"]
+    level = levels[min(len(levels) - 1, verbose)]  # capped to number of levels
```
Comment (neutrinoceros) [inline nonos/logging.py:26]: I think this first part is actually a `parse_verbose_level` function. 🤔
Following that, `setup_logging` becomes a mere wrapper for `logging.basicConfig` and can be removed.

Comment (volodia99) [reply nonos/logging.py:26]: You suggest I keep this first part in a `parse_verbose_level` function which returns the level, and I write the `logging.basicConfig` directly in the main, with the level returned by `parse_verbose_level` ?

**Pull request merged**